plugins/rapture/nexus-rapture-plugin/src/main/java/org/sonatype/nexus/rapture/internal/security/SessionAuthenticationFilter.java

```diff
@@ -52,6 +52,19 @@
 
   public static final String P_REMEMBER_ME = "rememberMe";
 
+  public static final String DELETE_METHOD = "DELETE";
+
+  /**
+   * Allow access, by default rules, and also if this is a request to logout.
+   */
+  @Override
+  protected boolean isAccessAllowed(final ServletRequest request,
+                                    final ServletResponse response,
+                                    final Object mappedValue)
+  {
+    return super.isAccessAllowed(request, response, mappedValue) || isLogoutRequest(request, response);
```

Comment:
unsure what you mean, though FTR I think this can be simplified more to just allow access if the request is to logout (method=DELETE).   probably again the shiro framework here makes this a lot more confusing that really needed.